pkg/products/rhsso/reconciler.go

```diff
@@ -486,11 +486,11 @@ func getUserDiff(keycloakUsers []keycloak.KeycloakAPIUser, openshiftUsers []user
 	return added, deleted
 }
 
-func syncronizeWithOpenshiftUsers(ctx context.Context, keycloakUsers []keycloak.KeycloakAPIUser, serverClient k8sclient.Client, ns string, installation *integreatlyv1alpha1.RHMI) ([]keycloak.KeycloakAPIUser, error) {
+func syncronizeWithOpenshiftUsers(ctx context.Context, keycloakUsers []keycloak.KeycloakAPIUser, serverClient k8sclient.Client, ns string, installation *integreatlyv1alpha1.RHMI, logger l.Logger) ([]keycloak.KeycloakAPIUser, error) {
 	var openshiftUsers *usersv1.UserList
```

Comment:
WDYT if we clean this up a bit? It looks like it's the remains from not fully reverted multitenant work (same for line 491)

Reply:
I am not sure what you are asking.

Reply:
There is no point in declaring openshiftUsers and error as vars since the value will be assigned in line https://github.com/integr8ly/integreatly-operator/pull/2279/files#diff-ea0e91bd9f2b961ab416fa3821815794ee1f0a283090fff9c72af19d81a42b72R493 and nowhere else. 
I would remove code from 490 and 491